<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(47, 47, 41);
font: 75 10pt &quot;Yu Gothic&quot;;
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="drawingArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>513</width>
      <height>513</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHilbertCurve">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 119, 92);</string>
    </property>
    <property name="text">
     <string>Hilbert Curve</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxOrder">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 128, 255);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Order:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColor">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 119, 92);</string>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 119, 92);</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(105, 79, 255);</string>
    </property>
    <property name="text">
     <string>21132_Shreyash</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(105, 79, 255);</string>
    </property>
    <property name="text">
     <string>A10 Hilbert Curve</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxX1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxY1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>180</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxX2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxY2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>185</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y2</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
